pyrogram/client.py

```diff
@@ -749,13 +749,12 @@ def load_plugins(self):
                     for name in handlers:
                         # noinspection PyBroadException
                         try:
-                            handler, group = getattr(module, name).handler
-
-                            if isinstance(handler, Handler) and isinstance(group, int):
-                                self.add_handler(handler, group)
+                            for handler, group in getattr(module, name).handlers:
+                                if isinstance(handler, Handler) and isinstance(group, int):
+                                    self.add_handler(handler, group)
 
-                                log.info('[{}] [LOAD] {}("{}") in group {} from "{}"'.format(
-                                    self.session_name, type(handler).__name__, name, group, module_path))
+                                    log.info('[{}] [LOAD] {}("{}") in group {} from "{}"'.format(
+                                        self.session_name, type(handler).__name__, name, group, module_path))
 
                                 count += 1
```

Comment:
The indentation here is still wrong; `count += 1` will also count anything that's not an handler.

Reply:
oh wait, logging

Reply:
i think it is solved, in wonder why in this Change Request you marked the logging lines

Reply:
No, I didn't, I selected the line containing `count += 1`, but you get a preview of the above code as well.

Reply:
ah, that's confusing 
![image](https://user-images.githubusercontent.com/49692607/111610571-2a7c1400-87e4-11eb-982e-1aa9206d0947.png)